includes/filerepo/file/LocalFile.php

```diff
@@ -1072,18 +1072,7 @@ public function getMetadataItems( array $itemNames ): array {
 	 * @return string|Blob
 	 */
 	public function getMetadataForDb( IReadableDatabase $db ) {
-		$this->load( self::LOAD_ALL );
```

Comment:
Suggestion: We might switch to filtering metadata when displaying them. This approach will be easier to modify in the future in case we want to keep / remove some metadata tags. Also it would make https://fandom.atlassian.net/browse/SPLAT-191 obsolete

Suggestion: Use hooks so our changes could be accepted upstream